tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "rootDir": "src",
     "outDir": "dist",
     "sourceMap": true,
-    "declaration": true
+    "declaration": true,
+    "strict": true
```

Comment:
I would not use this field because it can change between TS versions - I would rather add them one by one to guarantee safe updates